examples/mini-hlint/src/Main.hs

```diff
@@ -99,7 +99,9 @@ main = do
       let flags = defaultDynFlags fakeSettings fakeLlvmConfig
       case parse file flags s of
         POk _ m -> analyzeModule flags m
-        PFailed _ ->
-          -- putStrLn $ showSDoc flags $ pprLocErrMsg $ mkPlainErrMsg flags loc err
-          putStrLn "Fix me" -- to do; call `getErrorMessages` to replace the old code above
+        PFailed s -> sequence_
+          [ putStrLn msg
+          | msg <- showSDoc flags <$>
```

Comment:
I'd have moved the `showSDoc` so it read `[ putStrLn $ showSDoc flags msg | msg <- ...]`. The use of `<$>` as map is unusual.

Reply:
Yes, that's nice too. That said, are you ok with keeping the current phrasing?

Reply:
Going with the following which eliminates the `map`.
```
PFailed s -> sequence_
          [ putStrLn (showSDoc flags msg)
          | msg <- pprErrMsgBagWithLoc (snd $ getMessages s flags)
          ]
```